### PR TITLE
Adjusts wizard spells and familiars for balance and usability

### PR DESCRIPTION
**Pull request by ggermanicus**
- Wizard begins play with a free summoning stone in their bag.
- Spell speed upgrades no longer lie to you about being instant.
- Bugfixes with stones for successful ghost-getting and fiend/infiltrator clothing selection.
- Summoning stones no longer have a chance to go to maint where no one but the wizard could have used them anyways?
- Simplemobs that can already ventcrawl can now ventcrawl while having things. (This is necessary for a familiar.)
- Transformations can be toggled.
- Transformations recharge only when you un-transform.
- Transformed mobs keep their faction.

Several familiars adjusted:
- Familiars inherit the master's faction.
- Fiend: Firebolt rate of fire is 1.5 seconds.
- Familiar/shapeshifter: Mouse replaced with possum. All animals get spells - Pike gets to summon carp, cats have Knock, Invisibility, and Exude Pleasantness, bears have regeneration and Exchange Wounds... and possums get Summon Random Items.
- Champion: Laser armor buffed. Armor uses a modified version of urist's knight_yellow, to reflect its robustness.
Screenshot 2023-10-13 034303

- Fiend and Infiltrator will allow you to select already-present gendered clothing.
- Overseer grim robe and hood can no longer be removed by other people.
- Caretaker spell Trance has a 5 minute upper limit.

Wizard spell adjustments:
- Dyrnwyn did pitiable amounts of burn damage. Now it is at least conceivable to use at 30 damage unupgraded, 40 upgraded.
- Subjugate will only affect mobs the wizard can see and disables people half as long.
- Summon Mirage did not summon mirages, and if it did, that would be useless. You now summon space carp.
- Mastercrafted Armor now has merc voidsuit level armor, and raider level siemen's.
- Summon Space Bats summons more bats, which have less hp than carp, and the spell has a shorter cooldown than summoning carp.
- Sacrifice self-damage has been doubled.
- Corrupt Form changes your shape depending on power upgrades. Power 2 gets a faithless with double the health. Power 3 gets a juggernaut+50 hp, with a 5 minute cooldown and no flashdark aura.
- Ethereal Jaunt no longer includes upward-downward movement, has a 30 second minimum cooldown, and you can't drop items while jaunting.
- Passage cooldown reduced from 25 to 10 seconds.
- Adds summoning paper to Summon Random Items.

### CODE REVIEW

File: code/modules/spells/aoe_turf/summons.dm [hunk 1]
```diff
@@ -53,20 +53,18 @@
...
-
 /spell/aoe_turf/conjure/mirage/empower_spell()
 	if(!..())
 		return 0
 
 	summon_amt++
 
-	return "You now summon [summon_amt] mirages per spellcast."
\ No newline at end of file
+	return "You now summon [summon_amt] mirages per spellcast."
```
Comment (scrdest) [inline code/modules/spells/aoe_turf/summons.dm:70]:
```suggestion
	return "You now summon [summon_amt] carps per spellcast."
```

File: code/modules/spells/general/toggle_armor.dm [hunk 2]
```diff
@@ -71,14 +71,14 @@
...
-	invocation = "thrusts /his hand forward, and it is enveloped in golden embers!"
+	invocation = "thrusts a hand forward, and it is enveloped in golden embers!"
 	armor_pieces = list(/obj/item/excalibur = slot_r_hand)
 	hud_state = "excalibur"
 
 /spell/toggle_armor/fiend
 	name = "Toggle Armor (Fiend)"
 	invocation_type = SpI_EMOTE
-	invocation = "snaps /his fingers, and /his clothes begin to shift and change"
+	invocation = "snaps theirfingers, and their clothes begin to shift and change"
```
Comment (scrdest) [inline code/modules/spells/general/toggle_armor.dm:81]:
```suggestion
	invocation = "snaps their fingers, and their clothes begin to shift and change"
```

File: code/modules/spells/targeted/equip/dyrnwyn.dm [hunk 3]
```diff
@@ -33,8 +33,8 @@
 	if(!..())
 		return 0
 
-	material = MATERIAL_SILVER
-	return "Dyrnwyn has been made pure: it is now made of silver."
+	material = MATERIAL_PLASTEEL
+	return "Dyrnwyn has been made pure: it is now made of plasteel."
```
Comment (scrdest) [inline code/modules/spells/targeted/equip/dyrnwyn.dm:37]: This message doesn't make sense for plasteel.

File: code/modules/spells/general/toggle_armor.dm [hunk 4]
```diff
@@ -106,15 +106,15 @@
...
+	invocation = "flicks their wrists, one at a time"
 	armor_pieces = list(/obj/item/stamp/chameleon = slot_l_hand,
 						/obj/item/pen/chameleon = slot_r_hand)
 	hud_state = "forgery"
 
 /spell/toggle_armor/overseer
 	name = "Toggle Armor (Overseer)"
 	invocation_type = SpI_EMOTE
-	invocation = " is enveloped in shadows, before /his form begins to shift rapidly"
+	invocation = " is enveloped in shadows, form shifting rapidly"
```
Comment (Glloyd) [inline code/modules/spells/general/toggle_armor.dm:117]:
```suggestion
	invocation = " is enveloped in shadows, their form shifting rapidly"
```

**Review by Glloyd — CHANGES_REQUESTED** [review]

**Comment by Glloyd** [comment]
it's wizard time

**Pull request merged**